Representable/representable
# Hash created on coi submission and stored in table with entry ID

string used for hash:

![image](https://user-images.githubusercontent.com/15281033/103430889-82688e80-4b96-11eb-8877-9b506277fa4b.png)

![image](https://user-images.githubusercontent.com/15281033/103430895-8a283300-4b96-11eb-8394-e5d77f4a840b.png)

to test:
add key in environ
create entry
go to postico
check for signature and entry id in the main) signature table
